package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vott-react",
-    "version": "0.2.0",
+    "version": "0.2.2",
```

Comment:
Are we planning on having a way of automating versioning? Or are we saving that for later?

Reply:
This PR actually removes the automatic patch versioning. Discussed with Wallace this morning and he prefers to keep it manual, since only we will know if it's a patch, a minor, or a major change.

Reply:
Makes sense. Thanks for clarifying!